### DISCUSSION TITLE
mibs error22

### DISCUSSION THREAD

**Discussion by llshsw** [prometheus/snmp_exporter#1329] [Q&A]
make generator mibs
make: 'generator' is up to date.
>> Downloading HPE to /tmp/tmp.koK8f7BMOp
>> Downloading LIEBERT_GP_PDU.MIB to /tmp/tmp.lXK4ZWF0jN
curl: (22) The requested URL returned error: 403 
make: *** [Makefile:378: mibs/LIEBERT_GP_PDU.MIB] Error 22

Does anyone know how to solve this error?

**Comment by bastischubert** [comment]
hi @llshsw 
since you've not provided any further details about your environment i can only guess that it was as temporary error. have you tried again? if so does the error still exist?

**Comment by Aankhen** [comment]
For what it’s worth, I got some 403s during <kbd>make generator mibs</kbd> right now, but waiting and retrying after a small interval would let it continue. (It happened for more than one URL.)

**Comment by TafkaMax** [comment]
One solution is  this, make it actually retry curl requests on 4XX errros: https://github.com/prometheus/snmp_exporter/pull/1665

Works for my CI that generates my mibs.
